frontend/storage/reducers/breadcrumb.ts

```diff
@@ -0,0 +1,17 @@
+//Until we use pageReducer it seems like esLint won't count PageState and UpdatePageAction as used
+// eslint-disable-next-line no-unused-vars
+import {AppStore, SetBreadcrumbAction, SET_BREADCRUMB} from '../common';
+const initialState: AppStore = {
+	headerBreadcrumb: 'Home',
+};
+export function breadcrumbReducer(state = initialState, action: SetBreadcrumbAction): AppStore {
+	switch (action.type) {
+		case SET_BREADCRUMB: {
+			return {
+				headerBreadcrumb: action.headerBreadcrumb || 'Home',
```

Comment:
Someday you'll have more than one variable in the state and this code will cause you a headache or two. Change it to:

```
return {
   ...state,
    headerBreadcrumb: <copy the rest of the line>
};
```

[The spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) will essentially make this happen:
 1. It'll populate the new object with all of the properties of the old one (Essentially create a shallow-copy)
 2. Replace `headerBreadcrumb` with the new value.

You can opt to not do this change at the moment but you'll save yourself some time in the future by immediately adding it.